apps/omg_api/lib/application.ex

```diff
@@ -51,6 +53,42 @@ defmodule OMG.API.Application do
              }
            ]}
       },
+      %{
+        id: :in_flight_exit,
+        start: {
+          OMG.API.EthereumEventListener,
+          :start_link,
+          [
+            %{
+              # TODO check if synced_height_update_key is appropriate
+              synced_height_update_key: :last_exiter_eth_height,
+              service_name: :in_flight_exit,
+              block_finality_margin: deposit_finality_margin,
+              get_events_callback: &OMG.Eth.RootChain.get_in_flight_exit_starts/2,
+              process_events_callback: &OMG.API.State.in_flight_exits/1,
+              get_last_synced_height_callback: &OMG.Eth.RootChain.get_root_deployment_height/0
+            }
+          ]
+        }
+      },
+      %{
+        id: :piggyback,
+        start: {
+          OMG.API.EthereumEventListener,
+          :start_link,
+          [
+            %{
+              # TODO check if synced_height_update_key is appropriate
+              synced_height_update_key: :last_exiter_eth_height,
+              service_name: :piggyback,
+              block_finality_margin: deposit_finality_margin,
```

Comment:
`block_finality_margin` is wrong in both additions. You must ensure `inflights` and `piggybacks` get processed after deposits (see `exiter` below). Dry the "`+ 1`" bit into a variable `exiters_finality_margin` and don't remember to move the comment `we need to be just one block after...` accordingly!